### DISCUSSION TITLE
Load dynamic data

### DISCUSSION THREAD

**Discussion by antoinechamot** [openfga/community#46] [Q&A]
- In microservice architecture. How to modify data model dynamically. Do i need to make rest call from various services to the autorization api to update the data ? or there is another way ?

**Comment by aaguiarz** [comment]
Hi @antoinechamot,

Why do you need to modify the data model dynamically? 

You can create a new model using [this api]( https://openfga.dev/api/service#/Authorization%20Models/WriteAuthorizationModel). That endpoint returns an `authorization_model_id` that you can specify when calling the rest of the endpoints (check, read, expand, etc) to indicate that's the model you want to use (if not, it will use the latest version).

If what you are asking for is how to add tuples dynamically, you can do that using the [write API](https://openfga.dev/api/service#/Relationship%20Tuples/Write)

Let me know if that helps
